src/components/Navbar/index.module.css

```diff
@@ -24,6 +35,10 @@
   position: absolute;
   top: $size-64;
   z-index: 1;
+
+  width: 140px;
```

Comment:
I'm totally ok with fixed `px` values, just name them in vars so we can understand what they do.